src/genie/libs/parser/iosxe/show_platform.py

```diff
@@ -479,7 +479,7 @@ def cli(self, output=None):
         p16 = re.compile(r'^\s*[Ll]icense +[Ll]evel\: +(?P<license_level>.+)$')
 
         # entservices   Type: Permanent
```

Comment:
Please add the comment like below
```suggestion
        # entservices   Type: Permanent
        # License Level: entservices   Type: Permanent Right-To-Use
```